alfresco-transform-core-aio/alfresco-transform-core-aio-boot/src/main/java/org/alfresco/transformer/AIOCustomConfig.java

```diff
@@ -32,13 +32,31 @@
 import org.alfresco.transformer.transformers.MiscAdapter;
 import org.alfresco.transformer.transformers.PdfRendererAdapter;
 import org.alfresco.transformer.transformers.TikaAdapter;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Primary;
 
 @Configuration
+@ComponentScan
 public class AIOCustomConfig
 {
+    @Value("${libreoffice.executor.path}")
+    private String libreofficePath;
+
+    @Value("${pdf_renderer.executor.path}")

```

Comment:
Very minor, but lets change this to pdfrenderer as it is already like that in some other places